handler_error.go

```diff
@@ -227,11 +226,9 @@ func (e *ErrorHandler) HandleError(w http.ResponseWriter, r *http.Request, errMs
 		if prior, ok := r.Header["X-Forwarded-For"]; ok {
 			clientIP = strings.Join(prior, ", ") + ", " + clientIP
 		}
-		ip = clientIP
 	} else {
 		log.WithFields(logrus.Fields{
 			"prefix":      "gateway",
-			"user_ip":     ip,
```

Comment:
Even if this ip is always empty string?

I can have it be `"user_ip": "",` if that is still useful.